io-sim-classes/src/Control/Monad/Class/MonadSTM/Strict.hs

```diff
@@ -50,7 +50,10 @@ data StrictTVar m a = StrictTVar
    , tvar      :: !(Lazy.LazyTVar m a)
    }
 
--- TODO: it can be removed after 'Server.Socket' is re-written using 'MonadSTM'
```

Comment:
Removed this TODO, because this will actually be useful also in the `NoUnexpectedThunks` instance, so let's just leave it in.